glitch/apps.py

```diff
@@ -49,11 +71,16 @@ def glitch_video(input_path: str, output_path: str) -> None:
     width, height = get_video_size(input_path)
     reader = start_ffmpeg_reader(input_path)
     writer = start_ffmpeg_writer(output_path, width, height)
+
+    min_effect_length = int(options['min_effect_length']) or 1
```

Comment:
this will not handle properly the default values: `options['max_effect_length']` will throw a `KeyError` exception. If you want to get a `None` if the key doesnt exists you should use `get`. The correct way to do this (if the contents of `options` are str that we want to cast would be 
```python
    max_effect_length = int(options.get('max_effect_length', 15))
```
using `or` to handle default values have the problem that would use the default value when the received one is `0` (0 is evaluated as `False`, then `0 or x` returns `x`


```